PyInstaller/depend/analysis.py

```diff
@@ -133,6 +133,68 @@ def _reset(self, user_hook_dirs):
         self._hooks_pre_safe_import_module = self._cache_hooks('pre_safe_import_module')
         self._hooks_pre_find_module_path = self._cache_hooks('pre_find_module_path')
 
+        # Search for run-time hooks in all hook directories.
+        self._available_rthooks = defaultdict(list)
+        for uhd in self._user_hook_dirs:
+            uhd_path = os.path.join(uhd, 'rthooks.dat')
```

Comment:
Why? I don't see that this would affect the operation of the code -- whether `uhd` is relative or absolute, `open` will still look for it. Am I missing something?

Reply:
On second thought, I added it.